src/redis-cli.c

```diff
@@ -746,7 +746,7 @@ static int versionIsSupported(sds version, sds since) {
         versionPos = strchr(versionPos, '.');
         sincePos = strchr(sincePos, '.');
         if (!versionPos || !sincePos)
-            return 0;
+            return 1;
```

Comment:
this change only affects cases that one of the versions is only 2 digits, right?
i.e. 8.0 instead of 8.0.0?

maybe we should return 1 only in case both are NULL (and not when one is NULL and the other isn't)?
or better yet, replace the `atoi` with `0` like `versionPos? atoi(versionPos): 0;

Reply:
This is also happening for 3 digits for both `since` and `version`.
I will fix the code that for a not-equal number of digits, the return value will be `0` (Not supported) (although it probably can't happen)